app/controllers/resource_controller.rb

```diff
@@ -100,6 +110,7 @@ def destroy
   private
 
   def search_resources
+    resource_class
```

Comment:
where is this used / tested?

Reply:
tested by new test in resouce_controller_test ... the default index response is "all resources"